### ISSUE TITLE
External html imports not working with 0.4.1

### ISSUE THREAD

**Issue by mkazlauskas** [Polymer/polymer-build#64]
### Description

External html imports not working with 0.4.1
### Versions & Environment
- polymer-build: 0.4.1
- node: 6.1.0
- Operating System: Arch x64
#### Steps to Reproduce

`<link rel="import" href="https://...">`
`$ polymer build`
#### Expected Results

No error is thrown
#### Actual Results

`Uncaught exception: Error: ENOENT: no such file or directory`
### Fix

_get-dependencies-from-document.ts#getDependenciesFromDocument_
Change line: `if (!next.href) {`
With: `if (!next.href || isDependencyExternal(next.href)) {`

**Comment by FredKSchott** [comment]
Sorry I can't reproduce. We are covering a lot of this logic in our tests already. I've added the following line to `test-project/bower_components/loads-external-dependency.html` to explicitly test your example and everything is still passing during build: 
```
<link rel="import" href="https://www.example.com/import.html">
```

Can you write and share a test that is failing, or share some steps that I could reproduce?

**Comment by mkazlauskas** [comment]
Are you sure you're testing 0.4.1? `master` doesn't even have the file that I suggested to fix.

**Comment by FredKSchott** [comment]
Ah, this must have been fixed with the new analyzer. We'll try to get this out in a v0.5.0 release asap

**Comment by FredKSchott** [comment]
Fixed in v0.5.0!
